.gitmodules

```diff
@@ -1,24 +1,8 @@
-[submodule "llvm"]
-	path = compiler
-	url = https://github.com/RadeonOpenCompute/llvm.git
-	branch = amd-common
-[submodule "lld"]
-	path = lld
-	url = https://github.com/RadeonOpenCompute/lld.git
-	branch = amd-common
-[submodule "clang"]
-	path = clang
-	url = https://github.com/RadeonOpenCompute/hcc-clang-upgrade.git
-	branch = clang_tot_upgrade
-[submodule "compiler-rt"]
-	path = compiler-rt
-	url = https://github.com/RadeonOpenCompute/compiler-rt
-        branch = amd-common
 [submodule "rocdl"]
 	path = rocdl
 	url = http://github.com/RadeonOpenCompute/ROCm-Device-Libs.git
         branch = master
-[submodule "clang-tools-extra"]
-	path = clang-tools-extra
-	url = https://github.com/RadeonOpenCompute/clang-tools-extra.git
-	branch = amd-common
+[submodule "llvm-project"]
+	path = llvm-project
+	url = https://github.com/david-salinas/llvm-project
+	branch = amd-common-monorepo-hcc
```

Comment:
is this supposed to be amd-stg-open-hcc?

Reply:
Nvm, you mentioned that in your commit. So the project isn't live yet

Reply:
yes.  I was just using this repo/branch to validate that HCC could be built with a monorepo.  The "real" PR will have this correct, which will be github.com/llvm-project and branch amd-stg-open-hcc, when that branch is available.